### PR TITLE
updates

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -135,17 +136,27 @@ def printCombatHP(charHP, enemyName, enemyHP): #A function defined for the Hitpo
...
+                time.sleep(3)
                     
         else:
             print("")
             print("Unfortunately you missed!")
-            time.sleep(2)
+            time.sleep(3)
+
+            enemyHitChance = random.uniform(0.1, 1.0)
+            enemyHitChance >= hitChance
```
Comment (JakeChampion) [inline app.py]: What's this line meant to do?

Comment (PeterChampion) [inline app.py]: Generate a random decimal number between 0.1 and 1.0 and check if that value is larger than or equal to 0.4 which is what is required to hit.

Comment (JakeChampion) [inline app.py]: The results of this boolean operation is not being used anywhere. It is a pointless operation to execute.

File: app.py [hunk 2]
```diff
@@ -174,27 +185,27 @@ def printCombatHP(charHP, enemyName, enemyHP): #A function defined for the Hitpo
...
         enemyHitChance = random.uniform(0.1, 1.0)
         if enemyHitChance >= hitChance:
-            enemyWarriorDmg = random.randint(1,4)
-            charHP -= enemyWarriorDmg
-            print("")
-            print("The", enemyWarrior, "hits!")
-            print("")
-            print("The", enemyWarrior, "strikes you dealing", enemyWarriorDmg, "damage!")
-            time.sleep(2)
+                enemyWarriorDmg = random.randint(1,4)
```
Comment (JakeChampion) [inline app.py:202]: too much whitespace?

**Pull request merged**